run-checks

```diff
@@ -334,6 +334,35 @@ if [ "$STATIC" = 1 ]; then
       echo "$got"
       exit 1
     fi
+
+    echo "Checking installation of golangci-lint"
+    GOLANGCI_INSTALL_DIR="/tmp"
+    GOLANGCI_LINT="$GOLANGCI_INSTALL_DIR/golangci-lint"
+    # Ensure GOLANGCI_VERSION matches the golangci-lint version in
+    # .github/workflows/test.yaml (without leading v)
+    GOLANGCI_VERSION="1.51.2"
+    if ! command -v $GOLANGCI_LINT > /dev/null || ! $GOLANGCI_LINT --version | grep -q $GOLANGCI_VERSION ; then
+        if command -v curl > /dev/null ; then
```

Comment:
is it possible to install it by using goinstall function?
also there is a snap for golangci-lint

Reply:
Good idea, goinstall might be a simpler option here, since it can get a particular version more easily than the snap. I'm worried about the snap auto updating and causing `run-checks` to fail without there being changes to snapd code.